lara-typescript/src/section-authoring/util/move-utils.tsx

```diff
@@ -44,14 +48,15 @@ export const moveSection = (args: IMoveSectionSignature) => {
       pageIndex: destPageIndex
     } = findSectionAddress(pages, destSectionId));
   }
-  else if (destPageId) {
-    destPageIndex = pages.findIndex(p => p.id === destPageId);
+
+  destPageIndex = pages.findIndex(p => p.id === destPageId);
+  if (destSectionIndex == null) {
+    destSectionIndex = (pages.find(p => p.id === destPageId)?.sections.length) || 0;
   }
 
   // We must have a destination page:
-  if (!(destPageIndex)) {
-    error(`can't find destination ${destination}`);
-    return false;
+  if (destPageIndex == null) {
```

Comment:
I think this should be `if ((destPageIndex === null) || (destPageIndex === -1)) {` since you are using `findIndex` and not `find`.